scripts/travis/03-download-thunder-composer.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Can we rename `download-thunder-*` scrips into `build-thunder-*`? Because we are actually building the project there.